controller/backup_volume_controller.go

```diff
@@ -314,10 +314,20 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 		log.Infof("Found %d backups in the backup target that do not exist in the cluster and need to be pulled", count)
 	}
 	for backupName := range backupsToPull {
+		backupLabelMap := map[string]string{types.GetLonghornLabelKey(types.LonghornLabelVolumeAccessMode): string(longhorn.AccessModeReadWriteOnce)}
+
+		backupURL := backupstore.EncodeBackupURL(backupName, backupVolumeName, backupTargetClient.URL)
+		if backupInfo, err := backupTargetClient.BackupGet(backupURL, backupTargetClient.Credential); err == nil {
+			backupLabelMap[types.GetLonghornLabelKey(types.LonghornLabelVolumeAccessMode)] = backupInfo.Labels[types.GetLonghornLabelKey(types.LonghornLabelVolumeAccessMode)]
```

Comment:
Ideally, it should be null check, but in go, it will return empty string if the key does not exist, so it's fine.

Reply:
Modified, it would check key exists first.